src/pages/City/components/NearestCitiesWeatherWidget/NearestCitiesWeatherWidget.tsx

```diff
@@ -21,7 +22,7 @@ const NearestCitiesWeatherWidget = ({ loading, cityName }: Props) => {
     }
 
     if (cityName) {
```

Comment:
**Suggestion** Add data to if statement instead of cityName `if(data)`. CityName is passed as parameter so this statement is always true right now. If you change it like I suggested, you can then delete `?` in return statement.